### DISCUSSION TITLE
how to generate config with teams token ?

### DISCUSSION THREAD

**Discussion by willstore69** [rany2/warp.sh#2] [Q&A] [answered]
im Tryng ./warp.sh -T <token> but script not send me wgcf config

is this how to used ?
./warp.sh -4 -T <token>

**Comment by rany2** [comment, marked as answer]
It explains it in the help page:

Regarding Teams enrollment:
  1. Visit `https://<teams id>.cloudflareaccess.com/warp`
  2. Authenticate yourself as you would with the official client
  3. Check the source code of the page for the JWT token or use the following code in the "Web Console" (Ctrl+Shift+K):
  	  `console.log(document.querySelector("meta[http-equiv='refresh']").content.slice(64))`
  4. Pass the output as the value for the parameter -T. The final command will look like:
  	  `./warp.sh -T eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.....`
